webrtc.html

```diff
@@ -457,18 +457,18 @@
         </div>
         <p>An example array of RTCIceServer objects is:</p>
         <pre class="example highlight"><code>[
-     { "urls": "stun:stun1.example.net" },
-     { "urls": ["turns:turn.example.org", "turn:turn.example.net"],
-       "username": "user",
-       "credential": "myPassword",
-       "credentialType": "password" },
-     { "urls": "turns:turn2.example.net",
-       "username": "22BIjxU93h/IgwEb",
-       "credential": {
-                       "macKey": "WmtzanB3ZW9peFhtdm42NzUzNG0=",
-                       "accessToken": "AAwg3kPHWPfvk9bDFL936wYvkoctMADzQ5VhNDgeMR3+ZlZ35byg972fW8QjpEl7bx91YLBPFsIhsxloWcXPhA=="
+     { urls: "stun:stun1.example.net" },
+     { urls: ["turns:turn.example.org", "turn:turn.example.net"],
+       username: "user",
+       credential: "myPassword",
+       credentialType: "password" },
+     { urls: "turns:turn2.example.net",
+       username: "22BIjxU93h/IgwEb",
+       credential: {
+                       macKey: "WmtzanB3ZW9peFhtdm42NzUzNG0=",
+                       accessToken: "AAwg3kPHWPfvk9bDFL936wYvkoctMADzQ5VhNDgeMR3+ZlZ35byg972fW8QjpEl7bx91YLBPFsIhsxloWcXPhA=="
                      },
-       "credentialType": "oauth" },
+       credentialType: "oauth" },
```

Comment:
oh... do we want the dangling comma here (and elsewhere)?
I wonder if there is a way to extract the examples and run them through eslint...